src/applications/vaos/components/DateTimeSelectField.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+// import PropTypes from 'prop-types';
+import CalendarWidget from './calendar/CalendarWidget';
+import moment from 'moment';
+
+class DateTimeSelectField extends Component {
+  getOptionsByDate = selectedDate => {
+    const options = [];
+    const availableSlots = this.props.formContext.availableSlots;
```

Comment:
It seems like it's possible for `availableSlots` to be undefined which would cause the loop on the next line to blow up